evaluator/evaluator/app/routers/leaderboard.py

```diff
@@ -126,17 +127,6 @@ def get_ranking_key(leaderboard_entry: LeaderboardEntry):
         return list(leaderboard_entry.result.keys())[0]
 
 
-def get_skills(token: str) -> dict:
```

Comment:
Wenn ich das richtig sehe, kannst du https://github.com/nclskfm/square-core/pull/59/files#diff-62183a6268cadd0a2b726ea2f1a4a28a8ea958fb088816c3a0f2fc424b3adfd1L17 dann auch löschen in der Datei hier.

Also den Teil:
```python
SKILL_MANAGER_API_URL = os.getenv(
    "SKILL_MANAGER_API_URL", "https://square.ukp-lab.de/api/skill-manager"
)
```